docs_user/adoption-attributes.adoc

```diff
@@ -12,7 +12,7 @@ ifeval::["{build}" != "downstream"]
 :OpenShiftShort: OCP
 :ocp_curr_ver: 4.16
 :rhos_curr_ver: Antelope
-:rhos_prev_ver: Wallaby
+:rhos_prev_ver: 
```

Comment:
Do you need this attribute if it is not used in assemblies?

Reply:
@ciecierski Yes, I still need it because it is used downstream.

Reply:
Will do!